<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimSettings</class>
 <widget class="QDialog" name="SimSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia symulacji</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="3" column="0">
    <widget class="QSpinBox" name="refTimeB">
     <property name="specialValueText">
      <string>Tryb nieblokujący (jeżeli możliwy)</string>
     </property>
     <property name="suffix">
      <string> Hz</string>
     </property>
     <property name="prefix">
      <string>Częstotliwość odświeżania: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QDoubleSpinBox" name="simTimeB">
     <property name="specialValueText">
      <string>Czas symulacji bez ograniczeń</string>
     </property>
     <property name="prefix">
      <string>Czas symulacji: </string>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>86400.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="srateB">
     <property name="suffix">
      <string> Hz</string>
     </property>
     <property name="prefix">
      <string>Częstotliwość próbkowania: </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>simTimeB</tabstop>
  <tabstop>srateB</tabstop>
  <tabstop>refTimeB</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SimSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SimSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
